net/freeradius/src/opnsense/service/templates/OPNsense/Freeradius/users

```diff
@@ -0,0 +1,32 @@
+{% if helpers.exists('OPNsense.freeradius.general.enabled') and OPNsense.freeradius.general.enabled == '1' %}
+
+{% if helpers.exists('OPNsense.freeradius.user.users.user') %}
+{%   for user_list in helpers.toList('OPNsense.freeradius.user.users.user') %}
+{%     if user_list.enabled == '1' %}
+{{ user_list.username }}  Cleartext-Password := "{{ user_list.password }}"
+       Framed-Protocol = PPP,
+{%       if user_list.ip is defined %}
+       Framed-IP-Address = {{ user_list.ip }}{% endif %}{% if user_list.subnet is defined %},
+       Framed-IP-Netmask = {{ user_list.subnet }}{% endif %}{% if user_list.vlan is defined %},
+       Tunnel-Type = VLAN,
+       Tunnel-Medium-Type = IEEE-802,
+       Tunnel-Private-Group-Id = {{ user_list.vlan }}
```

Comment:
sicher: ich glaub ich hab da was falsch gesehen:

von der Einrückung her spannt sich das if in Zeile 8 bis zu Zeile 14 - was bedeuten würde, dass es `user_list.ip` für die Konfiguration von VLANs nötig wäre. Schaut aber eher so aus als hätte ich das falsch gesehen.